oauth2client/pkce.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2016 Google Inc. All rights reserved.
```

Comment:
I see, I'd still argue against it in this case. We're strictly a client (and we've done a bad job of just being a client).